source/conf.py

```diff
@@ -74,14 +74,15 @@
 
 prolog_template = string.Template(
     '.. _`SYCL Specification`: '
-    + 'https://www.khronos.org/registry/SYCL/specs/sycl-2020-provisional.pdf'
+    + 'https://registry.khronos.org/SYCL/specs/sycl-2020/html/sycl-2020.html'
     + """
 .. |true| replace:: ``true``
 .. |false| replace:: ``false``
 .. |2020| replace:: *Since SYCL 2020*
 .. _oneAPI:  https://oneapi.com
 .. _SYCL: https://www.khronos.org/sycl/
 .. |SYCL_SPEC| replace:: `SYCL Specification`_
+.. |SYCL_SPEC_HEADER_FILES| replace:: SYCL Specification `Section 4.3 <https://registry.khronos.org/SYCL/specs/sycl-2020/html/sycl-2020.html#sec:headers-and-namespaces>`__
```

Comment:
It would be better to avoid repeating this URL for every entry.  Can you define `https://registry.khronos.org/SYCL/specs/sycl-2020/html/sycl-2020.html` in a common variable, and then use that variable for all these entries?  That will make it easier if we ever change the URL for some reason.

Reply:
From our perspective two options on the table:
1. Define _https://registry.khronos.org/SYCL/specs/sycl-2020/html/sycl-2020.html_ in a common variable, and then use that variable for all these entries 
   pros -> easier change the URL in feature
   cons -> user must manually navigate/find appropriate sub-chapter 

2. Define URL for every entry :
    pros-> User easily open appropriate chapter and have clear view
    cons -> potential URL change allocate more effort for rework in conf.py

Could you check with Mike and choose option? If you have new proposal/recommendation that is welcome

Reply:
I was thinking that you could do it like this: b0e0ca9564dbb6cfa7d20bd1c9204c3578a04750.

This has the advantages of both your options with none of the drawbacks.